src/lambda_codebase/initial_commit/bootstrap_repository/adf-bootstrap/deployment/example-global-iam.yml

```diff
@@ -23,4 +23,40 @@ Resources:
             Resource:
               - "*"
       Roles:
-        - adf-cloudformation-deployment-role
\ No newline at end of file
+        - adf-cloudformation-deployment-role
+
+  # ADFTerraformRole:
+  #   # This is the role that will be used to deploy Terraform resources from within 
+  #   # the target account. You should scope this policy depending on what you would 
+  #   # like to deploy within certain Organizational Units.
+  #   # NOTE: below is a sample IAM policy. This policies should NOT be used for purposes other than testing.
+  #   # Uncomment the `ADFTerraformRole` and `ADFTerraformPolicy` resources if you want to enable Terraform support in ADF.
+  #   Type: AWS::IAM::Role
+  #   Properties:
+  #     RoleName: "adf-terraform-role"
+  #     AssumeRolePolicyDocument:
+  #       Version: "2012-10-17"
+  #       Statement:
+  #         - Effect: Allow
+  #           Sid: "AssumeRole"
+  #           Principal:
+  #             AWS:
+  #               - !Sub arn:aws:iam::${AWS::AccountId}:role/adf-codebuild-role
+  #           Action:
+  #             - sts:AssumeRole
+  #     Path: /
+  # ADFTerraformPolicy:
```

Comment:
Can you add a comment above this policy that states that this policy is only required when you want to deploy to the deployment account using Terraform. If they need to push to other accounts, they don't need to add this policy.

Reply:
Added details to the description